Neos.Flow/Classes/Mvc/ActionRequest.php

```diff
@@ -494,7 +494,7 @@ public function getControllerActionName(): string
         $controllerObjectName = $this->getControllerObjectName();
         if ($controllerObjectName !== '' && ($this->controllerActionName === strtolower($this->controllerActionName))) {
             $controllerClassName = $this->objectManager->getClassNameByObjectName($controllerObjectName);
-            $lowercaseActionMethodName = strtolower($this->controllerActionName) . 'action';
+            $lowercaseActionMethodName = $this->controllerActionName . 'action';
```

Comment:
removing the `strtolower` is related to psalm??

Reply:
Yes.

```
ERROR: RedundantFunctionCall - Packages/Framework/Neos.Flow/Classes/Mvc/ActionRequest.php:497:42 - The call to strtolower is unnecessary (see https://psalm.dev/280)
            $lowercaseActionMethodName = strtolower($this->controllerActionName) . 'action';
```

And it's true, because that block is only run if  `$this->controllerActionName === strtolower($controllerActionName)`…

Reply:
Wow. Psalm is cleverer than me o.O